Spec/RealtimeClientPresence.swift

```diff
@@ -904,7 +904,7 @@ class RealtimeClientPresence: QuickSpec {
                             guard let error = error else {
                                 fail("error expected"); done(); return
                             }
-                            expect(error.message).to(contain("Channel denied access based on given capability"))
+                            expect(error.message).to(contain("Invalid clientId for credentials"))
```

Comment:
That looks like your test is doing something odd with the `clientId` and changing this error message only indicates that this test is no longer testing "should result in an error if the client does not have required presence permission".  Can you enable verbose logging and send us the log to look at, I suspect you're sending an invalid `clientId` somewhere